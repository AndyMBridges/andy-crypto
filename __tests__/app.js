import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

// tested endpoint
import App from '../src/components/app.js';

configure({ adapter: new Adapter() });

describe('App page', () => {
  test('shows title', () => {
    const app = shallow(<App />);

    expect(app.find('h1').text()).toEqual('Andy Crypto');
  });

  test('matches snapshot', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
