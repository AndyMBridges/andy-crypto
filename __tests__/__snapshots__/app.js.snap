// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App page matches snapshot 1`] = `
<div
  className="container"
>
  <header
    className="header"
  >
    <a
      href="#/"
      onClick={[Function]}
    >
      <h1
        className="header__title"
      >
        Andy Crypto
      </h1>
    </a>
  </header>
  <div
    className="container__wrap container__wrap--home"
  >
    <div
      className="content is-loading"
    >
      <div
        className="panel"
      />
      <div
        className="loader"
      >
        <div
          className="loader__icon"
        />
      </div>
    </div>
  </div>
</div>
`;
